core/src/main/kotlin/model/Documentable.kt

```diff
@@ -5,7 +5,7 @@ import org.jetbrains.dokka.transformers.descriptors.KotlinTypeWrapper
 import org.jetbrains.dokka.links.DRI
 import org.jetbrains.dokka.pages.PlatformData
 
-class Module(val packages: List<Package>) : Documentable() {
+class Module(override val name: String?, val packages: List<Package>) : Documentable() {
```

Comment:
Correct me if I'm wrong, but do we ever create `Module` with nulled name? Because I failed to find this situation, and if so, we do not need a nullability here.

Reply:
Changed.